src/payment/strategies/humm/humm-payment-strategy.ts

```diff
@@ -0,0 +1,59 @@
+import { FormPoster } from '@bigcommerce/form-poster';
+
+import { CheckoutStore,  InternalCheckoutSelectors } from '../../../checkout';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError } from '../../errors';
+import PaymentActionCreator from '../../payment-action-creator';
+import { PaymentRequestOptions } from '../../payment-request-options';
+import { ExternalPaymentStrategy } from '../external';
+
+export default class HummPaymentStrategy extends ExternalPaymentStrategy {
+
+    constructor(
+        _store: CheckoutStore,
+        _orderActionCreator: OrderActionCreator,
+        _paymentActionCreator: PaymentActionCreator,
+        _formPoster: FormPoster
+    ) {
+            super(
+            _store,
+            _orderActionCreator,
+            _paymentActionCreator,
+            _formPoster
+        );
+        }
+
+    async initialize(): Promise<InternalCheckoutSelectors> {
+        return this._store.getState();
+    }
+
+    async deinitialize(): Promise<InternalCheckoutSelectors> {
```

Comment:
similar here, this is not required, it's already implemented in the parent class and the logic here didn't change